src/send/EnterAmount.test.tsx

```diff
@@ -126,6 +126,7 @@ const defaultParams = {
   prepareTransactionError: undefined,
   onPressProceed: onPressProceedSpy,
   ProceedComponent: SendProceed,
+  origin: 'send' as const,
```

Comment:
Is this type cast needed?

Reply:
yes, otherwise TS infers it as a generic string.